cli/module_registry_test.go

```diff
@@ -0,0 +1,34 @@
+package cli
+
+import (
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	"go.viam.com/rdk/testutils"
+	"go.viam.com/rdk/testutils/inject"
+	"go.viam.com/test"
+)
+
+func TestUpdateModelsAction(t *testing.T) {
+	_, filename, _, ok := runtime.Caller(0)
+    test.That(t, ok, test.ShouldBeTrue)
```

Comment:
Looks like your editor is not running `gofmt`; let me know if you need any help setting that up (will have to format these go files you're editing).